### PR TITLE
Set namespace of gRPC template

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Addresses https://github.com/aspnet/AspNetCore/issues/8451. 

Tested locally via CLI and VS.

### CODE REVIEW

File: src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/Protos/greet.proto [hunk 1]
```diff
@@ -1,5 +1,7 @@
 syntax = "proto3";
 
+option csharp_namespace = "GrpcService_CSharp";
```
Comment (JamesNK) [inline src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/Protos/greet.proto]: This will be replaced with the project's namespace?

Comment (ryanbrandenburg) [reply src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/Protos/greet.proto]: Yeah, wouldn't this have to be [GrpcService-CSharp](https://github.com/aspnet/AspNetCore/blob/0d722cf6dd90e590f9f0adb7d4ea28d0f371c1a3/src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/.template.config/template.json#L19)?

Comment (JunTaoLuo) [reply src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/Protos/greet.proto]: Not sure exactly how the substitution works but [`GrpcService_CSharp`](https://github.com/aspnet/AspNetCore/blob/master/src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/Startup.cs#L10) works. I tested this locally to make sure the project's namespace is used.

**Review by ryanbrandenburg — APPROVED** [review]
You proved my one concern wrong, :shipit:.

**Review by shirhatti — APPROVED** [review]

**Pull request merged**